users-backend/users/domain/repository.py

```diff
@@ -42,8 +41,12 @@ class UsersRepository(AbstractUsersRepository):
     def __init__(self, session: Optional[AsyncSession] = None) -> None:
         self.session = session
 
-    async def get_users(self) -> Paginable[User]:
-        return Paginable(self.session, User)
+    async def get_users(self, paginate=True) -> Union[list[User], Paginable[User]]:
+        if paginate:
+            return Paginable(self.session, User)
+
+        users = (await self.session.execute(select(User))).scalars().all()
+        return users
```

Comment:
Why are we making a query that returns ALL users? We are talking about 1k or more

Reply:
because you told me to do so :D

Reply:
I told you that paginating in an internal api doesn’t make much sense

having a query that returns all users, what would be the point?

Reply:
Yeah, probably we don't need all the user... I guess I can pass a list of ids